extensions/scarb-cairo-test/tests/examples.rs

```diff
@@ -8,6 +8,7 @@ use test_for_each_example::test_for_each_example;
 fn cairo_test(example: &Path) {
     Command::new(cargo_bin("scarb"))
         .arg("cairo-test")
+        .arg("--workspace")
```

Comment:
I don't understand, what does this change?

Reply:
The workspaces example from https://github.com/software-mansion/scarb/tree/main/examples/workspaces does define the root package, so cairo-test called in workspace root would test only a single package (the root) without passing `--workspace` flag.